src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.macOS.cs

```diff
@@ -213,7 +213,8 @@ private NSStringAttributes GetAttributes()
 		{
 			var attributes = new NSStringAttributes();
 
-			var font = NSFontHelper.TryGetFont((float)FontSize, FontWeight, FontStyle, FontFamily);
+			var fontSize = (float)FontSize * FeatureConfiguration.Font.FontSizeRatio;
+			var font = NSFontHelper.TryGetFont(fontSize, FontWeight, FontStyle, FontFamily);
```

Comment:
I kinda feel like the scaling logic should be unified in `NSFontHelper`, even if right now it's seemingly trivial